src/styles/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
```

do it with `margin-bottom`
```suggestion
    margin-bottom: 16px;
```